tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/BundlePublishFullAdd.java

```diff
@@ -149,6 +149,9 @@ private IStatus _autoDeploy(IPath output) throws CoreException {
 				retval = LiferayServerCore.error("Unable to copy file to auto deploy folder", ioe);
 			}
 		}
+		else {
+			retval = LiferayServerCore.error("Unable auto deploy bundle " + output.toPortableString());
```

Comment:
Print out reason why we were unable to deploy.  In this case it's because the autoDeployPath doesn't exist.